### DISCUSSION TITLE
Sub Collections

### DISCUSSION THREAD

**Discussion by lowtechninja** [NUWCDIVNPT/stig-manager#516] [Ideas and Enhancements]
It might be nice to have sub-collections. The use case would be you have a site with different OEM and each would need access to their own systems and you would want to report across them all. This would also facilitate multi-tenancy if you needed it.

**Comment by cd-rite** [comment]
Hi @lowtechninja Thanks for the suggestion!
Since, in the end, it kind of comes down to being able to view reports that span different groups of Assets/STIGs, I'm wondering if your other suggestion of multi-collection reports would also satisfy this requirement.  Then you could create lots of Collections for whatever suits your needs, granting access as you see fit, and view a consolidated report that sums them all. 

Right now, there are a couple ways you could accomplish something similar with our current feature set...
- Using the Restricted grants, you could individually give access to just the Assets/STIGs you want particular team members to see. 
- You could also create a new Collection for a particular team, and give them whatever access they need (Maybe Full, so they can see everything but cant add new assets/stigs), and then you can transfer the assets back into your "main" collection when they are done.

Both of those may be cumbersome at the moment depending on your specific use case, though. 

We will keep this use case in mind when we are prioritizing our next feature implementation. Thanks!
